blocks/parser.js

```diff
@@ -37,16 +39,14 @@ export function getBlockAttributes( blockNode, blockSettings ) {
  * @return {Array}          Block list
  */
 export default function parse( content ) {
-	return grammarParse( content ).reduce( ( memo, blockNode ) => {
+	return grammarParse( content ).map( ( blockNode ) => {
 		const settings = getBlockSettings( blockNode.blockType );
```

Comment:
What do you think if we fallback here to a `blockType = "html"` (or tiny, generic or whatever, just pick one) whenever we get `falsy` settings, we could call, getBlockSettings again with the default block after that. 

This would allow us to leverage the `TinyMCE` block everyone is talking about (me included) as a fallback block.